apps/meteor/app/api/server/v1/users.ts

```diff
@@ -1394,6 +1402,46 @@ API.v1.addRoute(
 	},
 );
 
+const userRoutes = API.v1.get(
+	'users.getPublicRoles',
```

Comment:
I was thinking if this is the best "namespace" for this information, I was thinking if it could be under "permissions.*" namespace or somewhere else..

if we keep it here under "users.*" namespace maybe we could call it something like "users.listByRole" like we have "users.listByStatus" and "users.listTeams" .. wdyt?